SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/SpatialWorkerFlags/SpatialWorkerFlagsTest.cpp

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/SpatialWorkerFlags.h"
+
+#include "TestDefinitions.h"
+#include "WorkerFlagsTestSpyObject.h"
+
+#define SPATIALWORKERFLAGS_TEST(TestName) \
+	GDK_TEST(Core, SpatialWorkerFlags, TestName)
+
+namespace
+{
+	Worker_FlagUpdateOp Create_Worker_FlagUpdateOp(const char* name, const char* value)
```

Comment:
Even for static methods in a source file would prefer we used the normal function and variable name conventions.